src/main/java/net/masterthought/cucumber/Configuration.java

```diff
@@ -94,4 +97,19 @@ public File getEmbeddingDirectory() {
         return new File(getReportDirectory().getAbsolutePath(), Configuration.EMBEDDINGS_DIRECTORY);
 
     }
+
+    /**
+     * @return Patterns to be used to filter out tags in the 'Tags Overview' chart. Returns an empty list by default.
+     */
+	public Pattern[] getTagsToExcludeFromChart() {
+		return tagsToExcludeFromChart;
+	}
+
+	/**
+	 * Stores the regex patterns to be used for filtering out tags from the 'Tags Overview' chart
+	 * @param patterns Regex patterns to match against tags
+	 */
+	public void setTagsToExcludeFromChart(Pattern... patterns) {
```

Comment:
1. OK

2. if regex fails then rethrows https://github.com/damianszczepanik/cucumber-reporting/blob/master/src/main/java/net/masterthought/cucumber/ValidationException.java and that's it. I don't want to have to many methods in configuration